patches/minecraft/net/minecraft/client/renderer/entity/ItemRenderer.java.patch

```diff
@@ -76,6 +84,15 @@
              this.m_115152_(bufferbuilder, p_115177_ + 2, p_115178_ + 13, 13, 2, 0, 0, 0, 255);
              this.m_115152_(bufferbuilder, p_115177_ + 2, p_115178_ + 13, i, 1, j >> 16 & 255, j >> 8 & 255, j & 255, 255);
              RenderSystem.m_69478_();
+@@ -360,7 +_,7 @@
+             RenderSystem.m_69493_();
+             RenderSystem.m_69482_();
+          }
+-
++         net.minecraftforge.common.MinecraftForge.EVENT_BUS.post(new net.minecraftforge.client.event.RenderGuiItemDecorationsEvent.Post(p_115175_, p_115176_, p_115177_, p_115178_, p_115179_));
```

Comment:
Also here. Use a ForgeHooksClient callback,

Reply:
Also, just to shorten the patch a bit, instead of adding content to the empty line, add the new code line before or after the empty line.